tequilapi/endpoints/transactor.go

```diff
@@ -586,44 +583,6 @@ func (te *transactorEndpoint) SettleIntoStakeAsync(c *gin.Context) {
 	resp.WriteHeader(http.StatusOK)
 }
 
-// swagger:operation POST /transactor/token/{token}/reward Reward
-// ---
-// summary: Returns the amount of reward for a token
-// parameters:
-// - in: path
-//   name: token
-//   description: Token for which to lookup the reward
-//   type: string
-//   required: true
-// responses:
-//   200:
-//     description: Token Reward
-//     schema:
-//       "$ref": "#/definitions/TokenRewardAmount"
-//   500:
-//     description: Internal server error
-//     schema:
-//       "$ref": "#/definitions/ErrorMessageDTO"
-func (te *transactorEndpoint) TokenRewardAmount(c *gin.Context) {
```

Comment:
This is probably not safe to remove. @Guillembonet could maybe comment more.

I am pretty sure that apps use this to call transactor which proxies requests to affiliator to check for token rewards. This was in a way used to any token, affiliate, referral, whatever.

Reply:
yep not removable, let's talk on slack

Reply:
You are right, thanks. This endpoint also related to Affiliate program.